cmds/modules/netlightd/nft.go

```diff
@@ -16,6 +16,7 @@ func ensureHostFw(ctx context.Context) error {
 nft 'add table inet filter'
 nft 'add table arp filter'
 nft 'add table bridge filter'
+nft 'add table nat'
```

Comment:
I really think this entire file can completely replaced now with a `.nft` file basically rewrite this as a ruleset (similar to what we have under nft/rules.nft) 

It's way cleaner than executing individual commands